sdk/cosmos/ci.yml

```diff
@@ -63,8 +63,8 @@ pr:
       - sdk/cosmos/azure-cosmos-spark_3_2-12/pom.xml
       - sdk/cosmos/azure-cosmos-spark_3-1_2-12/pom.xml
       - sdk/cosmos/azure-cosmos-spark_3-2_2-12/pom.xml
-      - sdk/cosmos/azure-spring-data-cosmos/pom.xml
-      - sdk/cosmos/azure-spring-data-cosmos-test/pom.xml
+#      - sdk/cosmos/azure-spring-data-cosmos/pom.xml
+#      - sdk/cosmos/azure-spring-data-cosmos-test/pom.xml
 
```

Comment:
If we leave the azure-cosmos ci broken for now, do we still need to comment these out?